Disable eventfd support for now due to https://www.illumos.org/issues/9782

The patch can be dropped when build host includes the fix for the issue.

--- samba-4.8.4/lib/replace/wscript.1	2018-09-01 19:12:28.700621802 +0000
+++ samba-4.8.4/lib/replace/wscript	2018-09-01 19:12:41.338306501 +0000
@@ -540,8 +540,8 @@
     if conf.CONFIG_SET('HAVE_PORT_CREATE') and conf.CONFIG_SET('HAVE_PORT_H'):
         conf.DEFINE('HAVE_SOLARIS_PORTS', 1)
 
-    if conf.CHECK_FUNCS('eventfd', headers='sys/eventfd.h'):
-        conf.DEFINE('HAVE_EVENTFD', 1)
+#    if conf.CHECK_FUNCS('eventfd', headers='sys/eventfd.h'):
+#        conf.DEFINE('HAVE_EVENTFD', 1)
 
     conf.CHECK_HEADERS('poll.h')
     conf.CHECK_FUNCS('poll')
